draft-ietf-mls-architecture.md

```diff
@@ -242,8 +245,9 @@ Instructions are on that page as well.  Editorial changes can be
 managed in GitHub, but any substantive change should be discussed on
 the MLS mailing list.
 
-End-to-end security is a requirement for instant messaging systems and is
-commonly deployed in many such systems. In this context, "end-to-end" captures
+End-to-end security is a used in the vast majority of instant messaging systems,
+and also deployed in systems for other purposes such as calling and conferencing.
+In this context, "end-to-end" captures
```

Comment:
This is where we should reference https://datatracker.ietf.org/doc/draft-knodel-e2ee-definition/ instead of having our own definition

Reply:
This paragraph is not a definition. It is a statement of adoption.  The only aspect of e2e we specifically call out is that the operator of the messaging system can be considered an attacker. 

As for including this reference anyway, looking at the status, this has several Not Ready telechat ballot positions, so I am inclined to leave it out.